ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/port-channel-interfaces.j2

```diff
@@ -80,6 +80,33 @@ interface {{ port_channel_interface }}
 {%         if port_channel_interfaces[port_channel_interface].rt is arista.avd.defined %}
       route-target import {{ port_channel_interfaces[port_channel_interface].rt }}
 {%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.redundancy is arista.avd.defined %}
+      redundancy {{ port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.redundancy }}
+{%         endif %}
+{%         if port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.designated_forwarder_election is arista.avd.defined %}
+{%             set designated_forwarder_cli = "designated-forwarder election" %}
+{%             if port_channel_interfaces[port_channel_interface].evpn_ethernet_segment.designated_forwarder_election.algorithm is arista.avd.defined("preference") %}
```

Comment:
Please don't check for the specific algorithm as new algorithms might be supported in future or the default might be changed anytime.

Reply:
Since some of the other parameters are only valid for `preference`, you need to keep this check. Either we keep it as-is or we add another `else` that just outputs the value of `algoritm` to cater for any new options.

Reply:
Oh yeah, I missed reading the below lines. Yeah we need this check.